enterprise/frontend/src/embedding/data-picker/DataSelectorDatabaseSchemaPicker/DataSelectorDatabaseSchemaPicker.tsx

```diff
@@ -121,20 +110,20 @@ const DataSelectorDatabaseSchemaPicker = ({
   }
 
   return (
-    <AccordionList
+    <AccordionList<Item>
       id="DatabaseSchemaPicker"
       key="databaseSchemaPicker"
       className={CS.textBrand}
       hasInitialFocus={hasInitialFocus}
       sections={sections}
       onChange={({ schema }: any) => onChangeSchema(schema)}
       onChangeSection={handleChangeSection}
-      itemIsSelected={(schema: Schema) => schema === selectedSchema}
+      itemIsSelected={({ schema }) => schema === selectedSchema}
       renderSectionIcon={renderSectionIcon}
       renderItemIcon={() => <Icon name="folder" size={16} />}
       initiallyOpenSection={openSection}
       alwaysTogglable={true}
-      showSpinner={showSpinner}
+      showSpinner={(x) => "active" in x && x.active === false}
```

Comment:
Please rename `x` to something human-friendly.
Also in:
- `enterprise/frontend/src/embedding/data-picker/DataSelectorTablePicker/DataSelectorTablePicker.tsx`
- `frontend/src/metabase/query_builder/components/DataSelector/DataSelectorDatabaseSchemaPicker/DataSelectorDatabaseSchemaPicker.tsx`
- `frontend/src/metabase/query_builder/components/DataSelector/DataSelectorTablePicker/DataSelectorTablePicker.tsx`